src/main/java/com/stuypulse/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,136 @@
+package com.stuypulse.robot.subsystems;
+
+import java.util.Arrays;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.stuypulse.robot.Constants;
+import com.stuypulse.stuylib.network.SmartNumber;
+
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Shooter extends SubsystemBase {
+
+    // Shooter Motors
+    private CANSparkMax leftShooterMotor;
+    private CANSparkMax rightShooterMotor;
+    private CANSparkMax middleShooterMotor;
+
+    // Shooter Encoders
+    private CANEncoder leftShooterEncoder;
+    private CANEncoder rightShooterEncoder; 
+    private CANEncoder middleShooterEncoder;
+
+    // Feeder Motor
+    private CANSparkMax feederMotor;
+
+    // Hood Solenoid
+    private Solenoid hoodSolenoid;
+
+    // SpeedControllerGroup
+    private SpeedControllerGroup shooterMotors; 
+
+    // PIDController
+    private PIDController shooterController;
```

Comment:
You need to add a PID controller for the feeder, it will also be running all match at a set speed with a flywheel